packages/i18n/__demo__/index.html

```diff
@@ -4,9 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>i18n Translate Benchmark</title>
-    <link rel="stylesheet" href="/node_modules/@refinitiv-ui/demo-block/demo.css">
   </head>
   <style>
+    body {
+      visibility: visible !important;
+    }
```

Comment:
Why do we need this ?

Reply:
Theme loader in demo block will add this style to body tag when theme has loaded but we don't have theme loader anymore.